src/parse/mdast/index.ts

```diff
@@ -99,8 +99,7 @@ export class MarkdownParseState {
     tokens?.forEach((token) => {
       if (token.hidden) return;
       const handler = this.handlers[token.type];
-      if (!handler)
-        throw new Error(`Token type \`${token.type}\` not supported by tokensToMyst parser`);
+      if (!handler) return;
```

Comment:
Can we have a console.warn or console error here? I think it is helpful to get some context, and warnings or console errors will be written out to logging.